src/main/java/io/cryostat/discovery/CustomDiscovery.java

```diff
@@ -165,9 +163,7 @@ Response doV2Create(
 
             if (Target.find("connectUrl", target.connectUrl).singleResultOptional().isPresent()) {
                 return Response.status(Response.Status.BAD_REQUEST.getStatusCode())
-                        .entity(
-                                V2Response.json(
-                                        Response.Status.BAD_REQUEST, "Duplicate connection URL"))
+                        .entity(Map.of(Response.Status.BAD_REQUEST, "Duplicate connection URL"))
```

Comment:
Similar to above, this can just be the `"Duplication connection URL"` string, no need to create a Map.